src/app/trainee/revalidation-history.interface.ts

```diff
@@ -16,6 +16,23 @@ export interface IRecommendation {
   submittedBy: string;
   submissionStatus: string;
   comments: IComment[];
+  deferralReason?: string;
+  deferralDate?: Date;
+
+  // private String proposedOutcomeCode;
```

Comment:
is the commented out code WIP due to BE not being complete?

Reply:
Yeah, I just pasted the back-end properties here to bind after discussions with Uzair and Jay. These are the current JAVA bindings but may be revised. Will delete as appropriate in data-bind task

Reply:
removed - kept in notepad